catboost/python-package/catboost/_catboost.pyx

```diff
@@ -108,6 +108,8 @@ class _NumpyAwareEncoder(JSONEncoder):
             return obj.tolist()
         return JSONEncoder.default(self, obj)
 
+class TMultiRegressionMetric:
```

Comment:
Let's rename TMultiRegressionMetric MultiLabelCustomMetric here.
And in c++ too (TMultiLabelCustomMetric).

And enumerate all methods which needs to implement.